community/kernel/src/main/java/org/neo4j/kernel/impl/transaction/log/LogVersionUpgradeChecker.java

```diff
@@ -47,11 +47,8 @@ public static void check( LogTailScanner tailScanner, Config config ) throws Upg
             LogEntryVersion latestLogEntryVersion = tailScanner.getTailInformation().latestLogEntryVersion;
             if ( latestLogEntryVersion != null && LogEntryVersion.moreRecentVersionExists( latestLogEntryVersion ) )
             {
-                String message = String.format(
-                        "The version your upgrading to is using a new transaction log format. This is a non-reversible " +
-                                "upgrade and you wont be able to downgrade after starting. To allow upgrade, please set " +
-                                "configuration parameter \"%s=true\"",
-                        GraphDatabaseSettings.allow_upgrade.name() );
+                String message = "The version your upgrading to is using a new transaction log format. This is a " +
+                        "non-reversible upgrade and you wont be able to downgrade after starting";
```

Comment:
Also would be helpful to see the version (both existing as well as current), right?

Reply:
If you believe that numbers like -9 will make the users less confused we could add that.